.github/workflows/lint.yml

```
@@ -0,0 +1,29 @@
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+      - name: Cache Maven packages
+        uses: actions/cache@v3
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('apps/backend/pom.xml') }}
+      - name: Run Checkstyle
+        run: cd apps/backend && mvn checkstyle:check
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)